### ISSUE TITLE
Spinner doesn't working fine when calling "show" method next time

### ISSUE THREAD

**Issue by ghost** [icanzilb/SwiftSpinner#96]
Spinner doesn't working fine when I want to show it next time..
Can someone helping me please?

**Comment by jessiemblack** [comment]
You should be more specific in your answer. Your problem is unclear.

**Comment by robcecil** [comment]
I have seen this problem too @jessalbarian 

I use SwiftSpinner with RxSwift (through a code pattern called ActivityIndicator), and out side of it, and when I do:

SwiftSpinner.show(<text 1>, animate: true)
then
SwiftSpinner.hide()
then immediately
SwiftSpinner.show(<text 2>, animate: true)

The second spinner will not appear.

**Comment by DavidLari** [comment]
Have you tried it without the hide?

**Comment by robcecil** [comment]
Yes. It works properly without a hide() called between the two show() method calls.

That is not a good solution for me however. If I have two separate network calls, independent of each other, and non-deterministically sequenced, I need to sandwich each call with a pair of show/hide somehow.

**Comment by icanzilb** [comment]
You could try showing/hiding `SwiftSpinner.sharedInstance`directly, but I haven't tried that before not sure if it's a brilliant idea ... though, if it works in your case - why not :)

**Comment by sam199488** [comment]
I'm facing the same issue (hide between two show, the last show doesn't work)
@icanzilb what do you mean by "showing/hiding SwiftSpinner.sharedInstance directly" ?

**Comment by andreamazz** [comment]
Hey @sam199488 
I reckon this issue should be fixed by one of the latest PR (#122). 
Can you test against the master version of the lib?
